backend/portfolio42_api/models.py

```diff
@@ -1,3 +1,29 @@
+from datetime import date
 from django.db import models
+from django.core.serializers import serialize
+from django.core.serializers.json import DjangoJSONEncoder
+from django.contrib.auth.models import AbstractUser
 
-# Create your models here.
+class User(AbstractUser):
+	intra_username = models.CharField(max_length=30, unique=True)
+	intra_id = models.IntegerField(unique=True, db_index=True)
+	first_name = models.CharField(max_length=50)
+	last_name = models.CharField(max_length=50)
+	email = models.EmailField(max_length=130)
+	intra_url = models.CharField(max_length=200);
+	image_url = models.CharField(max_length=800);
+	bio = models.TextField(max_length=800)
+	updated_at = models.DateTimeField(auto_created=True, auto_now=True)
+	is_admin = models.BooleanField(default=False)
+
+	REQUIRED_FIELDS = ["email", "intra_id", "first_name", "last_name", 'intra_url']
+	USERNAME_FIELD = "intra_username"
+
+	def __str__(self):
+		return "@" + self.intra_username
+
+	def wasUpdatedToday(self):
+		return self.updated_at > date.yesterday()
+
+	def serialize(self):
+		return serialize('json', [self])[1:-1]
```

Comment:
Do we want to use Django's serializers? Or will this be converted to django-rest serializers in a future update?

Reply:
This is actually using the Django json default serializer, remember last time, we were looking at the Source Code, we had it explicitly but then we realized it is default so I deleted it.

Also in the future we want to change this, yes ;D
***

Source:
https://github.com/django/django/blob/8e2460d599aec95f8cfe514d3cc8acdd4ca4b1fb/django/core/serializers/json.py#L31